templates/preview.html

```diff
@@ -0,0 +1,134 @@
+{% extends 'layouts/_base.html' %}
+
+{% from "components/panel/_macro.njk" import onsPanel %}
+{% from "components/button/_macro.njk" import onsButton %}
+{% from "components/accordion/_macro.njk" import onsAccordion %}
+
+{% set save_on_signout = true %}
+
+{% set breadcrumbs = {
+  "ariaLabel": 'Back',
+  "itemsList": [
+    {
+      "url": url_for("questionnaire.get_questionnaire"),
+      "id": "top-previous",
+      "text": _("Back"),
+      "attributes": {
+        "data-ga": 'click',
+        "data-ga-category": 'Navigation',
+        "data-ga-action": 'Previous link click'
+      }
+    }
+  ]
+} %}
+
+{% block main %}
+
+  <h1 class="ons-u-mt-m">{{ _("Preview of the questions in this survey") }}</h1>
+
+  {% call onsPanel({
+    "classes": 'ons-u-mb-m'
+  }) %}
+    <p>{{ _("To answer these questions you need to <a href='{url}'>start survey</a>").format(url=url_for('questionnaire.get_questionnaire')) }}</p>
+  {% endcall %}
```

Comment:
This panel needs to be print hidden. You can add a class to hide when printing.

Reply:
v62.0.1 of design system added now so will use `ons-u-ph` to hide this.